testsuite/pangu_lib/testnet_commands/transaction_emitter.py

```diff
@@ -0,0 +1,39 @@
+from .create_testnet import SystemContext
+import pangu_lib.util as util
+from kubernetes import client
+from test_framework.logging import log
+
+
+def transaction_emitter_main(testnet_name: str, system_context: SystemContext):
+    pod: client.V1Pod = create_pod(testnet_name)
+    system_context.kubernetes.create_resource(pod, testnet_name)
+
+
+def create_pod(testnet_name: str) -> client.V1Pod:
+    container: client.V1Container = client.V1Container(
+        name=f"{testnet_name}-tx-emitter",
+        image="aptoslabs/tools:devnet_performance",
```

Comment:
this image should be configurable as well ;) maybe if it's not specified, you can inherit from the top level pangu config

Reply:
It'd be UTIL file imo, not the pangu config. Can't access the pangu config here.